src/BuildScriptGenerator/DotNetCore/DotNetCoreBashBuildSnippet.sh.tpl

```diff
@@ -1,25 +1,56 @@
+tmpOutputDir="/tmp/puboutput"
+zippedOutputFileName=oryx_output.tar.gz
```

Comment:
We probably need to come up with a naming convention for any folders or files that Oryx creates